src/ManageCourses.Api/Model/CourseQualification.cs

```diff
@@ -0,0 +1,11 @@
+namespace GovUk.Education.ManageCourses.Api.Model
+{
+    public enum CourseQualification
+    {
+        Qts,
```

Comment:
xml doc with full names
"Qualified Teacher Status"
while we still remember what they are...